net-irc/quassel/quassel-9999.ebuild

```diff
@@ -99,22 +99,22 @@ src_configure() {
 	local mycmakeargs=(
 		-DUSE_QT4=OFF
 		-DUSE_QT5=ON
-		-DWITH_BREEZE=OFF
-		-DWITH_WEBKIT=OFF
-		-DWITH_BREEZE_DARK=OFF
-		-DWITH_OXYGEN=OFF
-		-DEMBED_DATA=OFF
+		-DUSE_CCACHE=OFF
 		-DCMAKE_SKIP_RPATH=ON
+		-DEMBED_DATA=OFF
+		-DWANT_CORE=$(usex server)
+		-DWANT_QTCLIENT=$(usex X)
+		-DWANT_MONO=$(usex monolithic)
+		-DWITH_BUNDLED_ICONS=$(usex bundled-icons)
+		-DWITH_OXYGEN_ICONS=$(usex oxygen)
 		$(cmake-utils_use_find_package crypt QCA2-QT5)
 		$(cmake-utils_use_find_package dbus dbusmenu-qt5)
 		$(cmake-utils_use_find_package dbus Qt5DBus)
 		-DWITH_KDE=$(usex kde)
 		-DWITH_LDAP=$(usex ldap)
-		-DWANT_MONO=$(usex monolithic)
-		-DWANT_CORE=$(usex server)
 		$(cmake-utils_use_find_package snorenotify LibsnoreQt5)
+		-DWITH_WEBKIT=OFF
 		-DWITH_WEBENGINE=$(usex urlpreview)
-		-DWANT_QTCLIENT=$(usex X)
 	)
```

Comment:
I would prefer to keep the use flags in alphabetical order instead of the cmake parameter.

Reply:
Ah, I didn't see any pattern, looked pretty random to me :) So I tried to group more logically.

Better now?